WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -282,6 +282,19 @@ def refund_payment
     redirect_to edit_registration_path(registration)
   end
 
+  def entry_fee_for_selected_events
+    competition = competition_from_params
+    event_ids = params[:event_ids] ||= {}
+
+    @entry_fee_amount = competition.base_entry_fee
+    event_ids.each do |eventId|
+      @entry_fee_amount += competition.competition_events.find_by_event_id!(eventId).fee
+    end
+    render json: {
+      html: render_to_string(partial: 'entry_fee_amount'),
```

Comment:
The reason you can't call `format_money` directly here is because `format_money` is defined in `ApplicationHelper`, which I guess is automatically included for erb files, but you need to explicitly include it to access its methods within a controller. See how competitions controller does this here: https://github.com/jfly/worldcubeassociation.org/blob/959ee3281a13138671509d924d77e9584a54bc25/WcaOnRails/app/controllers/competitions_controller.rb#L4.